tests/models/abiotic/test_abiotic_model.py

```diff
@@ -9,7 +9,12 @@
 import xarray as xr
 from xarray import DataArray
 
-from tests.conftest import log_check, patch_bypass_setup, patch_run_update
+from tests.conftest import (
+    log_check,
+    patch_bypass_setup,
+    patch_run_update,
+    patch_static_config,
```

Comment:
Why is there all of the patching and checking of the static setup in the AbioticModel testing specifically? Is this just a convenient test case setup for running these checks or do these need to be rolled out to all model tests?

If it is the first - and this is really fussy - it makes the tests harder to follow for their primary purpose. It seems like it would be better to have a separate test file.

Reply:
Neither. The patches are also used in hydrology and litter, in addition to both abiotic models. 

The problem is that the data sources used in those models for testing purposes do not honour the requirements for bypassing the setup and running the update. Typically, these models are not static models and include some of the variables that they are meant to initialise or update in either the setup or update methods. So the checks run in `__init__` fail.

Maybe I should have flagged this up and request from everyone to amend their data used in tests, but since that was a lot of work and no one complained when I implemented this in the previous PR, I assumed it was OK.

Reply:
Ooooooh. Bugger. I think this is something we want to revisit because I don't think it helps the clarity of the testing, but perhaps not here.